bin/suitcss

```diff
@@ -63,9 +63,16 @@ program.parse(process.argv);
  * Settings.
  */
 
+function requireOrParse(configFile) {
+  var configPath = resolve(configFile);
+  var ext = path.extname(configPath);
+  var readFn = ext === '.js' ? require : fs.readJsonSync;
```

Comment:
imho it shows the purpose of the fn more clearly and since the dep on `fs-extra` is used elsewhere I thought 'hey, exactly what we want here' : )
but anyway: your call, I'll push an update in a minute!